CRM/Mosaico/Upgrader.php

```diff
@@ -120,10 +120,27 @@ public function upgrade_4704() {
       FOREIGN KEY (`msg_tpl_id`) REFERENCES `civicrm_msg_template`(`id`)
       ON DELETE SET NULL
     ');
+    
+    CRM_Core_Invoke::rebuildMenuAndCaches(TRUE);
+
+    return TRUE;
+  }
+  
+  /**
+   * Add menu for traditional mailing.
```

Comment:
not a big deal, but the comment should be updated to reflect the change

Reply:
Fixed.